Added Material Design

SamRTheDev opened the pull request:
- Added Material Design using resources from Google's Material Design website.
- Included examples of other material design websites.
- Added the material link to the main index.html

SamRTheDev committed:
First Commit

-Adding Material Design

SamRTheDev committed:
2nd Commit

SamRTheDev committed:
3rd

StuffBySpencer commented:
Hey, nice job so far! Looks like you forgot your `css` folder & your `script.js` file though!

SamRTheDev committed:
Fix

-Re-added JS file that I accidentally deleted.
- Removed the css link since I deleted the css folder in order to match the same structure as the minimalist and vaporwave entries.

SamPuckett commented:
> Hey, nice job so far! Looks like you forgot your `css` folder & your `script.js` file though!

I don't know why I deleted the script.js file 😅 The css folder I deleted in order to match the format of the minimalist and vaporwave entries

StuffBySpencer commented:
Looks great! Few tweaks I'll make on my end, nice job with the design. Thank you very much for contributing! :^)
